cmd/earthly/flags.go

```diff
@@ -217,9 +217,9 @@ func (app *earthlyApp) rootFlags() []cli.Flag {
 			Destination: &app.buildID,
 			Hidden:      true, // Internal.
 		},
-		&cli.IntFlag{
+		&cli.DurationFlag{
 			Name:        "server-conn-timeout",
-			Usage:       "The time to wait for the http client to establish a connection to the API server",
+			Usage:       "The time to wait in seconds for the http client to establish a connection to the API server",
```

Comment:
this usage seems out of date now.
```suggestion
			Usage:       "Earthly API server connection timeout value",
```

I have no idea if urfave adds on extra context regarding the supported unit suffixes, if not maybe add on something like

`values should end with ms, or s` (technically it could also be an `h` or longer :laughing: